testing/mock.ts

```diff
@@ -232,6 +239,7 @@
  * until real time is restored. You can control how time ticks forward with the
  * `tick` method on the `FakeTime` instance.
  *
+ * @example
```

Comment:
the `This module is browser compatible` at the bottom will be part of the example body